plugins/qlcplus_4.12.1/import.js

```diff
@@ -233,13 +226,202 @@ function getOflWheels(qlcPlusFixture) {
+  IntensityUV: () => getColorIntensityCap(`UV`),
+  IntensityIndigo: () => getColorIntensityCap(`Indigo`),
+  IntensityLime: () => getColorIntensityCap(`Lime`),
+  IntensityHue: () => ({ type: `Generic` }),
+  IntensitySaturation: () => ({ type: `Generic` }),
+  IntensityLightness: () => ({ type: `Generic` }),
+  IntensityValue: () => ({ type: `Generic` }),
+  NoFunction: () => ({ type: `NoFunction` }),
+  PositionPan: ({ panMax }) => getPanTiltCap(`Pan`, panMax),
+  PositionTilt: ({ tiltMax }) => getPanTiltCap(`Tilt`, tiltMax),
+  PositionXAxis: () => ({
+    type: `BeamPosition`,
+    horizontalAngleStart: `left`,
+    horizontalAngleEnd: `right`,
+    helpWanted: `Is this the correct direction? Can you provide exact angles?`
+  }),
+  PositionYAxis: () => ({
+    type: `BeamPosition`,
+    verticalAngleStart: `top`,
+    verticalAngleEnd: `bottom`,
+    helpWanted: `Is this the correct direction? Can you provide exact angles?`
+  }),
+  ColorRGBMixer: () => ({
+    // basically this is also Hue
+    type: `Generic`
+  }),
+  ColorCTOMixer: () => ({
+    type: `ColorTemperature`,
+    colorTemperatureStart: `default`,
+    colorTemperatureEnd: `warm`,
+    helpWanted: `Can you provide exact color temperature values in Kelvin?`
+  }),
+  ColorCTCMixer: () => ({
+    type: `ColorTemperature`,
+    colorTemperatureStart: `cold`,
+    colorTemperatureEnd: `warm`,
+    helpWanted: `Is this the correct direction? Can you provide exact color temperature values in Kelvin?`
+  }),
+  ColorCTBMixer: () => ({
+    type: `ColorTemperature`,
+    colorTemperatureStart: `default`,
+    colorTemperatureEnd: `cold`,
+    helpWanted: `Can you provide exact color temperature values in Kelvin?`
+  }),
+  SpeedPanSlowFast: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `slow`,
+    speedEnd: `fast`,
+    comment: `only Pan`
+  }),
+  SpeedPanFastSlow: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `fast`,
+    speedEnd: `slow`,
+    comment: `only Pan`
+  }),
+  SpeedTiltSlowFast: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `slow`,
+    speedEnd: `fast`,
+    comment: `only Tilt`
+  }),
+  SpeedTiltFastSlow: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `fast`,
+    speedEnd: `slow`,
+    comment: `only Tilt`
+  }),
+  SpeedPanTiltSlowFast: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `slow`,
+    speedEnd: `fast`
+  }),
+  SpeedPanTiltFastSlow: () => ({
+    type: `PanTiltSpeed`,
+    speedStart: `fast`,
+    speedEnd: `slow`
+  }),
+  ColorMacro: () => ({
+    type: `ColorPreset`,
+    helpWanted: `Which color can be selected at which DMX values?`
+  }),
+  ColorWheel: () => ({
+    type: `WheelSlot`,
+    slotNumber: 1,
+    helpWanted: `Which color can be selected at which DMX values?`
+  }),
+  GoboWheel: () => ({
+    type: `WheelSlot`,
+    slotNumber: 1,
+    helpWanted: `Which gobo can be selected at which DMX values?`
+  }),
+  GoboIndex: () => ({
+    type: `WheelRotation`,
```

Comment:
Should be `WheelSlotRotation`

Reply:
Fixed in [`8415977`](https://github.com/OpenLightingProject/open-fixture-library/pull/978/commits/8415977d7ac67617a344ce778a11f7e768c712a8#diff-83e5c40e8119c68f3a2416ac07053e96R369)